djangoproject/static/js/mod/search-key.js

```diff
@@ -11,9 +11,8 @@ define([
         init: function(){
             var self = this;
             $(document).ready(function () {
-                $(window).keypress(function(e) {
-                    if ($('input:focus, textarea:focus').length === 0 &&
-                            e.which === 47) {  // The slash is 47.
+                $(window).keydown(function(e) {
+                    if ($('input:focus, textarea:focus').length === 0 && e.key === 'k' && e.ctrlKey) {
```

Comment:
rearrange to check the cheap comparisons on event attribute first, before doing a DOM query

```suggestion
                    if (e.key === 'k' && e.ctrlKey && $('input:focus, textarea:focus').length === 0) {
```